packages/cms-lib/__tests__/http.js

```diff
@@ -18,6 +40,89 @@ describe('http', () => {
     getAndLoadConfigIfNeeded.mockReset();
     getPortalConfig.mockReset();
   });
+  describe('getOctetStream()', () => {
+    it('makes a get request', async () => {
+      getAndLoadConfigIfNeeded.mockReturnValue({
+        httpTimeout: 1000,
+        portals: [
+          {
+            id: 123,
+            apiKey: 'abc',
+          },
+        ],
+      });
+      getPortalConfig.mockReturnValue({
+        id: 123,
+        apiKey: 'abc',
+      });
```

Comment:
I'd movie these into a `before()` section since they are shared across all tests in the `describe` block.